### ISSUE TITLE
building on Xcode 7 archive results in intermittent failure with creating symlink

### ISSUE THREAD

**Issue by mkagita** [CocoaPods/CocoaPods#4566]
I seem to be getting random failure with cocoapods project when compiling with archive option with xcodebuild (xcodebuild archive)

Error looks like below where it happens intermittently ...

/bin/ln -sfh  /build/UninstalledProducts/iphoneos/libApp 1.a build/Release-iphoneos/libApp 1.a
ln: /build/Release-iphoneos/libApp 1: File exists

when using xcodebuild command line with build (xcodebuild build ...), there is no problem.

Any ideas? Is it a problem with xcodebuild?

**Comment by segiddins** [comment]
Can you share a project that reproduces this issue?

**Comment by mkagita** [comment]
@segiddins  I posted the project at https://github.com/mkagita/PodIssue  You will need to replace some parameters in the build.sh to match the iOS provision you have. It's intermittent so it happens every 5 - 10 times executing the script. I suspected it had something to do with parallel compilation where we are trying to write to same file at the same time? Not sure.... 

I also have a line where I build Pods project because I'm experiencing https://github.com/CocoaPods/CocoaPods/issues/4567 
thanks for you help!
